src/routes/FilterInterface.svelte

```diff
@@ -0,0 +1,71 @@
+<script lang="ts">
+
+	import { RecursiveTreeView, type TreeViewNode } from '@skeletonlabs/skeleton';
+  import { nodes } from '../treeContent';
+    
+let myTreeViewNodes: TreeViewNode[] = nodes;
+let checkedNodes : string[] = ['javascript'];
+let indeterminateNodes : string[] = ['programming', 'language'];
+											
+</script>
+
+<div class="dark-container w-screen flex pt-10 flex-col">
+  <h2 class ="h2 pl-16" style="text-align: left;" >Virtueller Katalog</h2>
+  <div class="side_by_side w-screen pl-2 pt-5 flex">
+    <div class="tree w-1/2 box-border text-sm">
+      
+      <RecursiveTreeView selection multiple relational {nodes} bind:checkedNodes={checkedNodes} bind:indeterminateNodes />
+      
+    </div>
+    <div class="search w-1/2 border-box">
+      <form class='form'>
```

Comment:
```suggestion
      <form>
```
form doesn't apply any styles?